File: Python/Comp sci club/warmup1.cpp
// warmup1.cpp

#include <iostream>
#include <limits>

using namespace std;

int main() {
	int a, min_p = numeric_limits<int>::max();

	cin >> a;

	for (int i=0; i<a/2; i++) {
		if (a % i == 0 && (2*i) + (2*(a/i)) < min_p) {
			min_p = (2*i) + (2*(a/i));
		}
	}

	cout << min_p << endl;

	return 0;
}

File: Python/Comp sci club/warmup1.py
# warmup1.py

a = int(input())

min_p = 10000000

for i in range(1, a//2):
	if a % i == 0:
		if (2*i) + (2*(a/i)) < min_p:
			min_p =  (2*i) + (2*(a/i))

print(int(min_p))